configs/SPRACINGH7EVO/config.h

```diff
@@ -113,7 +113,8 @@
 #define MAG_I2C_INSTANCE        (I2CDEV_2)
 #define BARO_I2C_INSTANCE       (I2CDEV_2)
 
-#define USE_MPU_DATA_READY_SIGNAL
+// no define matching USE_MPU_DATA_READY_SIGNAL in Betaflight 4.5 - did this mean gyro_exti?
+// #define USE_MPU_DATA_READY_SIGNAL
```

Comment:
@hydra should this line be changed to define the gyro_exti pin, or should it be simply commented out, as we have done here?

Reply:
We have `USE_MAG_DATA_READY_SIGNAL`

Reply:
Ah... indeed, it could be that

Reply:
After some more digging found https://github.com/betaflight/betaflight/commit/671f8abbc57d93fdceade12ab2fdbc8e3997ac2f

So it can be removed as no longer used.